pkg/config/config_test.go

```diff
@@ -35,14 +55,49 @@ func TestLoadConfig(t *testing.T) {
 
 	t.Run("Non-existent file with fallback", func(t *testing.T) {
 		t.Parallel()
+		fileContents := `# Configuration for attributing commits with emails to GitHub user profiles
+# Used during codeowners generation.
+# List the emails associated with the given username.
+# The commits associated with these emails will be attributed to
+# the username in this yaml map. Any number of emails may be listed.
+attribution:
+  brandonroberts:
+    - robertsbt@gmail.com
+  jpmcb:
+    - john@opensauced.pizza
+  nickytonline:
+    - nick@nickyt.co
+    - nick@opensauced.pizza
+  zeucapua:
+    - coding@zeu.dev`
+
 		tmpDir := t.TempDir()
 		fallbackPath := filepath.Join(tmpDir, ".sauced.yaml")
-		require.NoError(t, os.WriteFile(fallbackPath, []byte("key: fallback"), 0644))
+		require.NoError(t, os.WriteFile(fallbackPath, []byte(fileContents), 0644))
+
+		// Print out the contents of the file we just wrote
+		writtenData, err := os.ReadFile(fallbackPath)
+		require.NoError(t, err)
+		fmt.Printf("Contents written to file:\n%s\n", string(writtenData))
```

Comment:
nit: usually in Go, you won't see printing to stdout during tests: the test should either pass or error in an intelligent way that helps the developer know what broke